app/templates/frameworks/agreement.html

```diff
@@ -0,0 +1,149 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}{{ framework.name }} Framework Agreement– Digital Marketplace{% endblock %}
+
+{% block breadcrumb %}
+{%
+with items = [
+    {
+        "link": "/",
+        "label": "Digital Marketplace",
+    },
+    {
+        "link": url_for(".dashboard"),
+        "label": "Your account",
+    },
+    {
+        "link": url_for(".framework_dashboard", framework_slug=framework.slug),
+        "label": "Your {} application".format(framework.name),
+    }
+]
+%}
+{% include "toolkit/breadcrumb.html" %}
+{% endwith %}
+{% endblock %}
+
+{% block main_content %}
+  {% if supplier_framework.agreementReturned %}
+  {% with
+     heading = "Your document has been uploaded",
+     messages = [
+      "You’ll be notified once your framework agreement has been countersigned by the Crown Commercial Service."
+     ]
+    %}
+      {% include "toolkit/temporary-message.html" %}
+    {% endwith %}
+  {% endif %}
+
+    <div class='column-two-thirds large-paragraph'>
+        {% with
+            heading = "Sign your " + framework.name + " framework agreement",
+            smaller = True,
+            with_breadcrumb = True
+        %}
+            {% include "toolkit/page-heading.html" %}
+        {% endwith %}
+    </div>
+
+    <div class='column-two-thirds large-paragraph'>
+        <div class="padding-bottom-small">
+            <p>Your agreement will need to be signed by both you and the Crown Commercial Service (CCS) 
+                before you can sell {{ framework.name }} services.</p>
+        </div>
+
+        <div class="padding-bottom framework-agreement-section-first">
+            <h3>1. Download your framework agreement</h3>
+            {% with
+                items = [
+                    {
+                    "file_type": "PDF",
+                    "link": url_for('.download_agreement_file', framework_slug=framework.slug, document_name='framework-agreement.pdf'),
+                    "title": "Download framework agreement (PDF, 216KB)"
+                    },
+                ]
+            %}
+            {% include "toolkit/documents.html" %}
+            {% endwith %}
+        </div>
+        
+        <div class="padding-bottom framework-agreement-section">
+            <h3>2. Read your framework agreement</h3>
+            <p>If you have a question about your framework agreement, <a href="#">contact CCS</a>.</p>
```

Comment:
This should probably have `rel="external"` to display an arrow thingy next to it.